.github/workflows/ci.yml

```diff
@@ -11,6 +11,11 @@ jobs:
 
     runs-on: self-hosted
 
+    strategy:
+      fail-fast: false
+      matrix:
+        baseos: [ centos, debian ]
```

Comment:
This is going to double the build time.
I think it should be implemented as a separate GHA flow to just to test Docker build.